perfkitbenchmarker/cloud_tpu.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import abc
+
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import resource
+
+
+flags.DEFINE_string('tpu_cidr_range', None, """CIDR Range for the TPU. The IP
```

Comment:
Is it reasonable to have a default here? Can you at least explain how to choose a value that works?

Reply:
No, this is a required flag in tpu creating command.